spec/mailers/user_mailer_spec.rb

```diff
@@ -1,19 +1,37 @@
 require "rails_helper"
 
 RSpec.describe UserMailer, type: :mailer do
-  subject(:mail) { UserMailer.sign_in_email(user, sign_in_link) }
   let(:user) { create(:user) }
 
-  let(:sign_in_link) do
-    Rails.application.routes.url_helpers.email_confirmation_url(
-      login_token: user.login_token, host: Rails.application.config.domain,
-    )
-  end
-
   describe "#sign_in_email" do
+    let(:sign_in_email) { UserMailer.sign_in_email(user, sign_in_link) }
+
+    let(:sign_in_link) do
```

Comment:
Are we using this link for anything?

Reply:
Not really. I just use it to call the mailer as in UserMailer.sign_in_email(user, sign_in_link) . I could pass in any string but since I have already created the link I just decided to keep it.